crime-commons-classes/src/main/java/uk/gov/justice/laa/crime/util/RequestBuilderUtils.java

```diff
@@ -12,6 +12,7 @@
 public class RequestBuilderUtils {
 
```

Comment:
As this util is used only for unit tests, can we move this class under /util/test/RequestBuilderUtils.java?